BUILD.bazel

```diff
@@ -42,12 +42,33 @@ FUEL_PATCH_VERSION = 0
 FUEL_PROJECT_NAME = "ignition-fuel-tools"
 fuel_library_location = "../ign_fuel_tools/libignition-fuel-tools4.so"
 
+GUI_MAJOR_VERSION = 4
```

Comment:
This seems a bit fragile, as it could break when underlying libraries change versions.

Is there any way to import this from the other packages?

Reply:
Probably, I haven't looked much into this yet